jsk_pcl_ros_utils/scripts/install_sample_data.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+from jsk_data import download_data
+
+
+def main():
+    PKG = 'jsk_pcl_ros_utils'
+
+    download_data(
+        pkg_name=PKG,
+        path='sample/data/2015-11-04-19-37-29_baxter-kiva-object-in-hand-cloud.tgz',
```

Comment:
We don't have to remove the file from install_test_data.py.
Following test needs it.
https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_pcl_ros_utils/test/pointcloud_to_pcd.test#L4
https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_pcl_ros_utils/test/pointcloud_to_point_indices.test